### ISSUE TITLE
Add a configurable mouse/keyboard shortcut to "Open Link In Split View"

### ISSUE THREAD

**Issue by insilications** [brave/brave-browser#42204]
### Platforms

all

### Description

Add a configurable mouse/keyboard shortcut to "Open Link in Split View" to eliminate the friction to open a link's context menu and click "Open Link in Split View" to open it in a split view.

**Comment by bsclifton** [comment]
cc: @hamirmahal

also cc: @simonhong @fallaciousreasoning

**Comment by fallaciousreasoning** [comment]
cc @rebron any ideas what we'd want this shortcut to be?

(on Linux, and probably Windows too)
- Ctrl + Click (open in new tab)
- Ctrl + Shift + Click (open in new tab)
- Shift + Click (open in new window)
- Alt + Click (download link)
- Meta + Click (nothing)


We could probably take Ctrl+Shift+Click or Alt+Shift+Click, but maybe Meta+Click is the way (not sure what the equivalent on OSX would be - CTRL+Click?)

**Comment by insilications** [comment]
Ctrl+Shift+Click, which is an easy-to-remember variation of Ctrl+Click (open in new tab) and Shift+Click (open in new window), seems like a good default choice to me.

**Comment by hamirmahal** [comment]
I'm actually not familiar with the "Open Link in Split View" feature.

Do we want to add a demo of what that looks like? It might help whoever ends up taking this ticket.

> Add a configurable mouse/keyboard shortcut to "Open Link in Split View" to eliminate the friction of having to open a link's context menu and click "Open Link in Split View" to open it in a split view.

I don't see it in this link's context menu.

![Image](https://github.com/user-attachments/assets/55ce480d-8e4a-4579-8853-85ecac18eeaa)

**Comment by insilications** [comment]
> I don't see it in this link's context menu.

The `brave-split-view` flag (brave://flags/#brave-split-view) is currently disabled in the stable channel (https://github.com/brave/brave-browser/issues/36986), as tracked by the https://github.com/brave/brave-browser/issues/38302 task. I am using the beta channel and I enabled the `brave-split-view` flag.

**Comment by fallaciousreasoning** [comment]
Yeah, it should show up when brave://flags/#brave-split-view is enabled in Nightly or Beta (and possibly also release now)

**Comment by insilications** [comment]
Any plans?

**Comment by fallaciousreasoning** [comment]
I think @simonhong is doing a bit of polish on the feature at the moment, so might come out in the current Nightly (1.75) but  we'll see 🤞

**Comment by insilications** [comment]
Any plans?

**Comment by ClaudiuHNS** [comment]
I see the option to set hotkey is there but can't assign the "Click" input to the hotkey combination. 

How can I add it so I can open links in split view pressing CTRL + SHIFT + LEFT_CLICK ? Its an incredibly useful feature being able to quickly check a link in without leaving the page or having to do it thru the context menu (especially on sites where context menu is disabled).

<img width="781" height="757" alt="Image" src="https://github.com/user-attachments/assets/67d40f1d-1535-4851-b7b5-0dc53ace8178" />

**Comment by insilications** [comment]
@ClaudiuHNS Unfortunately the `Open in split view` configurable shortcut here is just for when you select two tabs with `shift`. It will open the two selected tabs in split.

The Brave team must work on it @bsclifton @simonhong @fallaciousreasoning

**Comment by DamKast** [comment]
I totally agree with @ClaudiuHNS, this is an incredibly useful feature. It really helps to increase productivity. 
This feature is definitely one of the reasons why I am not yet switching from Arc to Brave. 
Arc is using the `Option + Click` shortcut to open a link in split view (on Mac), I really like it.

Also, to be honest I don't know who is using the download link feature (`Alt + Click` on windows or `Option + Click` on Mac that @fallaciousreasoning was mentioning).

Any ways, you guys can decide what key would work best for the shortcut. But it would be awesome.
😄  Thanks for your great work so far though 👍

**Comment by insilications** [comment]
This will be implemented in upstream, behind a feature flag:

https://issues.chromium.org/issues/456470504

cc @bsclifton @simonhong
